### PR TITLE
 Fix non-semantic newlines for `e.g. `, `i.e. `, `; @`

### LINKED ISSUE #791: Semantic newlines not used correctly in some multi-reference lists or after `e.g.`
`;` and `.` appear to have been triggers for a hard newline even when the `;` is in the middle of a reference list, or the `.` is part of e.g. `e.g.`.

### PR DESCRIPTION
**Pull request by michaelmhoffman**
Performed by

```
perl -0pi -e 's/e\.g\.\n/e.g. /g; s/i\.e\.\n/i.e. /g; s/;\n@/; @/g' *.md
```

Closes #791 
Refs https://github.com/greenelab/deep-review/issues/624

### CODE REVIEW

File: content/03.categorize.md [hunk 1]
```diff
@@ -28,25 +28,19 @@ In all cases that we examined, fewer than one million images were available for
...
 An alternative in the domain is to train towards human-created features before subsequent fine-tuning [@tag:Dhungel2016_mamm], which can help to sidestep this challenge though it does give up deep learning techniques' strength as feature constructors.
 
 A second strategy repurposes features extracted from natural images by deep learning models, such as ImageNet [@tag:Russakovsky2015_imagenet], for new purposes.
 Diagnosing diabetic retinopathy through color fundus images became an area of focus for deep learning researchers after a large labeled image set was made publicly available during a 2015 Kaggle competition [@tag:Pratt2016_dr].
-Most participants trained neural networks from scratch [@tag:Pratt2016_dr;
-@tag:Abramoff2016_dr; @tag:Leibig2016_dr], but Gulshan et al.
+Most participants trained neural networks from scratch [@tag:Pratt2016_dr; @tag:Abramoff2016_dr; @tag:Leibig2016_dr], but Gulshan et al.
 [@tag:Gulshan2016_dt] repurposed a 48-layer Inception-v3 deep architecture pre-trained on natural images and surpassed the state-of-the-art specificity and sensitivity.
```
Comment (dhimmel) [inline content/03.categorize.md]: can we delete newline preceding this sentence

**Comment by dhimmel** [comment]
I'd like to squash merge this, but I'm worried it will result change your authorship info to the email address that's not associated with GItHub. Can make sure the most recent commit in this PR uses the right authorship email address. You can set this manually with `git commit --author`

@michaelmhoffman in future, consider rebasing rather than merging to keep PR commit histories clean.

**Comment by dhimmel** [comment]
> I think we should merge and quickly re-attribute the commit if it doesn't squash as expected

Agreed. @agitter take the reigns.

> I think the authorship may be okay if we squash merge.

Interesting. I think it used to be author of most recent commit. Glad GH switched to this more sensible system.

@michaelmhoffman note that you are using a different email address for some of the the commits and you should investigate if you care about your commits being associated with your github account.

**Comment by agitter** [comment]
Squash merge behavior confirmed.  The authorship on 1db0c13e0d5828d325ac8e1eba523b77bc33a7d1 is correct.

**Pull request merged**